src/routes/blogs.rs

```diff
@@ -118,7 +114,7 @@ pub fn create(conn: DbConn, form: LenientForm<NewBlogForm>, user: User, intl: I1
             is_owner: true
         });
 
-        Ok(Redirect::to(uri!(details: name = slug.clone())))
+        Ok(Redirect::to(uri!(details: name = slug.clone(), page = _)))
```

Comment:
does this work to fill an Option with None? I did not know, and was so searching for something like that!

Reply:
Yes, this new syntax was introduced in the final release of Rocket 0.4 IIRC.

Reply:
how ok, missed that